From: thepapermen <the.paper.men@gmail.com>
Date: Tue, 24 Sep 2013 10:17:20 +0400
Subject: Fixed Django 1.6 incompatibility issue.

---
 guardian/shortcuts.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/guardian/shortcuts.py b/guardian/shortcuts.py
index 0ac75bd..11f49d1 100644
--- a/guardian/shortcuts.py
+++ b/guardian/shortcuts.py
@@ -394,7 +394,7 @@ def get_objects_for_user(user, perms, klass=None, use_groups=True, any_perm=Fals
         group_filters = {
             'permission__content_type': ctype,
             'permission__codename__in': codenames,
-            'group__%s' % get_user_model()._meta.module_name: user,
+            'group__%s' % get_user_model().groups.field.related_query_name(): user,
         }
         groups_obj_perms_queryset = group_model.objects.filter(**group_filters)
         if group_model.objects.is_generic():
-- 
